docs/using/query-analytics.md

```diff
@@ -216,5 +216,15 @@ In contrast, MongoDB uses the concept of a document where all essential informat
 
 Query Analytics can monitor MongoDB queries. Although MongoDB is not a relational database management system, you analyze its databases and collections in the same interface using the same tools.
 
+## Sharing a link to Query Analytics
+
+To share a link to Query Analytics use the *Copy Link* button. This will copy a link to the clipboard with all relevant information so when the link is opened it shows exactly as you were seeing. This will copy information such as the selected query, table page, selected filters, details tab and time range.
```

Comment:
```suggestion
To share a link for Query Analytics, use *Copy Link*. It copies the link to the clipboard with all the relevant information such as selected query, table page, selected filters, details tab, and time range. Thus, when you open the link, it will display all the exact information.
```